addon/-legacy-private/system/references/has-many.js

```diff
@@ -401,7 +401,8 @@ export default class HasManyReference extends Reference {
      @method reload
      @return {Promise} a promise that resolves with the ManyArray in this has-many relationship.
   */
-  reload() {
-    return this.hasManyRelationship.reload();
+  reload(options = {}) {
+    options.reload = true;
```

Comment:
we can delete the default value assignment and the `options.reload = true` assignment